app/helpers/application_helper.rb

```diff
@@ -1,5 +1,9 @@
 # Helps generate HTML constructs for object owned by the user
 module ApplicationHelper
+  def content_class_from_name(class_name)
```

Comment:
Reminder: We should grep for `.constantize` and use this method instead.

Reply:
We probably also want to throw a recognizable exception if `class_name` isn't a valid key.